core/core.go

```diff
@@ -217,10 +218,15 @@ func New(config Config) (*App, error) {
 	})
 
 	// Add custom contract addresses if needed.
+	var contractAddresses ethereum.ContractAddresses
+	var err error
```

Comment:
Is there a way to avoid doing this? The two `vars` in a row are pretty gross 🤢

Reply:
@jalextowle you can do something like this:

```
	// Add custom contract addresses if needed.
	var contractAddresses ethereum.ContractAddresses
	if config.CustomContractAddresses != "" {
		addresses, err := parseAndValidateCustomContractAddresses(config.EthereumChainID, config.CustomContractAddresses)
		if err != nil {
			return nil, err
		}
		contractAddresses = addresses
	} else {
		addresses, err := ethereum.NewContractAddressesForChainID(config.EthereumChainID)
		if err != nil {
			return nil, err
		}
		contractAddresses = addresses
	}
```

But honestly I prefer the way it's currently written. I don't see any problem with having two `var` declarations.

Reply:
Yeah, I'll just leave it. It's not too bad. I just wanted to see if there was some way that I could use `:=` here, but the scoping doesn't seem to work.